source/Minable.cpp

```diff
@@ -102,19 +102,19 @@ void Minable::Place(double energy, double beltRadius)
 	double curved = (pow(asin(theta * 2. - 1.) / (.5 * PI), 3.) + 1.) * .5;
 	theta = (eccentricity * curved + (1. - eccentricity) * theta) * 2. * PI;
 
-	// Now, pick the orbital "scale" such that, relative to the "belt radius":
-	// periapsis distance (scale / (1 + e)) is no closer than .4: scale >= .4 * (1 + e)
-	// apoapsis distance (scale / (1 - e)) is no farther than 4.: scale <= 4. * (1 - e)
-	// periapsis distance is no farther than 1.3: scale <= 1.3 * (1 + e)
-	// apoapsis distance is no closer than .8: scale >= .8 * (1 - e)
+	// Now, pick the orbital "orbitScale" such that, relative to the "belt radius":
+	// periapsis distance (orbitScale / (1 + e)) is no closer than .4: orbitScale >= .4 * (1 + e)
+	// apoapsis distance (orbitScale / (1 - e)) is no farther than 4.: orbitScale <= 4. * (1 - e)
+	// periapsis distance is no farther than 1.3: orbitScale <= 1.3 * (1 + e)
+	// apoapsis distance is no closer than .8: orbitScale >= .8 * (1 - e)
```

Comment:
Yes, but in this case I preferred to change that because `orbitScale` is the variable used [just 2 lines under](https://github.com/endless-sky/endless-sky/blob/8ed603dec83ef8337669e5787811b3a305ec7571/source/Minable.cpp#L112).

Would you me to change that?

Reply:
It says

> orbital "orbitScale"`

which is a bit weird. Change it back yeah

Reply:
Ah yeah: spotted :sweat_smile: